client/src/linkManager.js

```diff
@@ -306,6 +308,30 @@ export class linkManager extends LuigiClientBase {
     return this;
   }
 
+  /**
+   * Sends options to the route. The parameters are used by the `navigate` function. Use it optionally in combination with any of the navigation functions and receive it as part of the context object in Luigi Client.
+   * @memberof linkManager
+   * @param {Object} options
+   * @returns {linkManager} link manager instance
```

Comment:
```suggestion
   * @returns {linkManager} link manager instance
   * @since NEXTRELEASE
```
since its a new function